### ISSUE TITLE
Update Platform defect severity rubric and GitHub labels to be consistent with 508 Office audit ratings

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#56709]
## User Story
_"As a VFS team member, I want to understand the severity of VA 508 Office audit findings in the context of Platform's severity rubric, so that I know how to prioritize fixing issues."_

Assignee: @briandeconinck 
Peer Reviewer: @it-harrison 

## Description
When a VFS team goes through a staging review, they receive accessibility findings that are labeled according to the [Platform accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric). The rubric categorizes issues as:
- `508-defect-0` - Potentially Harmful (launch blocker, must be fixed immediately)
- `508-defect-1` - Critical (launch blocker, must be fixed before launch)
- `508-defect-2` - Serious (fix in 1-2 sprints post-launch)
- `508-defect-3` - Moderate (fix 1-3 sprints post-launch)
- `508-defect-4` - Trivial (consider fixing or exploring 2-4 sprints post-launch)

Note: There's some inaccuracy in the labeling here. Section 508 currently only requires conformance to WCAG 2.0, but Platform and the VA have adopted WCAG 2.1 as our standard. While there aren't huge differences between 2.0 and 2.1, the `508` part of the label may be read to imply a legal weight that's not totally accurate.

After launch, VFS teams are required to request an accessibility audit from the VA 508 Office. Those audits categorize issues according to the severity scale:
- Critical
- High
- Medium
- Low

Those defect levels are not defined, nor does the 508 Office recommend any timeframe for action.

@jeana-adhoc flagged this privately after her team received its first 508 Office report and had one Medium issue that was missed during the staging review. With the inconsistent vocabulary, the team wasn't certain if they were correctly mapping the 508 Office findings back to Platform guidance.

This is a small thing but it's an easy win for us to synchronize our vocabulary with the 508 Office. After some discussion with other accessibility specialists, we recommend:
1. Update the [Platform accessibility defect severity rubric](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric) to match the 508 Office vocabulary for defect levels 1 (Critical remains Critical), 2 (Serious becomes High), 3 (Moderate becomes Medium), and 4 (Trivial becomes Low).
   - Note that defect-0 findings are rare and --- hopefully! --- will always be found before they reach production and a 508 Office audit. There's no immediate need to match defect-0 with the 508 Office ratings.
2. Update the GitHub labels to remove the `508` part and to include the descriptive term.
   - `508-defect-0` becomes `a11y-defect-0 (Potentially Harmful)`
   - `508-defect-1` becomes `a11y-defect-1 (Critical)`
   - `508-defect-2` becomes `a11y-defect-2 (High)`
   - `508-defect-3` becomes `a11y-defect-3 (Medium)`
   - `508-defect-4` becomes `a11y-defect-4 (Low)`
3. For consistency, rename the `508/Accessibility` GitHub label as just `Accessibility`. This would also match other Governance practice areas.

## Impacted Artifacts
- [Platform accessibility defect severity rubric](https://vfs.atlassian.net/wiki/spaces/pilot/pages/1906671954/Accessibility+defect+severity+rubric)
- GitHub labels for `508-defect-0` through `508-defect-4` and `508/Accessibility`.
- GitHub [Accessibility Issue Ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=&labels=508%2FAccessibility&template=a11y-issue.yaml&title=508-defect-x%3A) (not owned by Governance, used by non-Governance a11y specialists)
- ~~Any Domo data sets pointing to these labels~~ (this work will be completed in another ticket)
- audit of any other platform website or Confluence pages that would be impacted by this change
   - [Staging Review findings - issue template](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2548957608/Staging+Review+findings+-+issue+template)
   - [Staging Review - VFS - Moderator Responsibilities](https://vfs.atlassian.net/wiki/spaces/PPS/pages/1819476255/Staging+Review+-+VFS+-+Moderator+Responsibilities)
   - [Anatomy of a Staging Review Issue ticket](https://vfs.atlassian.net/wiki/spaces/CC/pages/2060320997/Anatomy+of+a+Staging+Review+Issue+ticket)
   - [How to Document Findings for a Design System Staging Review](https://vfs.atlassian.net/wiki/spaces/PPS/pages/2572189697/How+to+Document+Findings+for+a+Design+System+Staging+Review)

## Tasks
- [x] Complete audit of internal and VFS-facing documentation that references the changing labels
   - Artifact list has been updated 
- [x] Reach out to other accessibility specialists to identify any other repositories using the same labels that will need to be updated
   - Slack message posted to private #a11y-specialist channel. All impacted repos identified:
      - va.gov-team
      - vets-design-system-documentation
      - va.gov-cms (will be updated by @laflannery when Platform updates the others)
- [x] Update label names in GitHub in all identified repositories
- [x] Update instances of label names throughout Confluence
   - [All draft documentation changes](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2636218438/56709+Drafts) 
- [x] Create change history post and draft messages to announce the label change
- [x] Update label names in Drafting Template (@shiragoodman)

## Peer Review
_To be completed by peer reviewer_
- [ ] User story and acceptance criteria are met

## Acceptance Criteria
- [ ] Defect severity rubric is consistent with VA 508 Office categories
- [ ] GitHub labels do not use the term "508"
- [ ] Internal and VFS-facing documentation uses new label names

### Refinement
- [x] Ticket has user story, description, tasks, and acceptance criteria
- [x] Ticket has [additional appropriate labels](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2220359751/WIP+Tech+Debt+Ticket+Organization), if applicable
- [x] Ticket has been sized 
- [x] Once all above items are checked, add the 'Ready' label.
### Planning
_If this ticket is picked up from the Backlog mid-sprint, connect with Shira to ensure the below items are completed correctly_
- [x] Ticket has been discussed as a team at Planning
- [x] Ticket has been assigned to appropriate initiative or quarterly epic
- [x] Ticket has been assigned to a Sprint
- [x] Ticket has been moved to the Planned pipeline in the Governance team board
- [ ] Ticket as an assignee and peer reviewer

**Comment by shiragoodman** [comment]
@briandeconinck I read through this ticket today. Thank you for writing it up and for recognizing the inconsistencies across the different governing bodies. 

One thing to consider, that you may want to bring to the larger a11y group, is if we really need 5 different defect levels. Trevor Pierce was responsible for creating the `508-defect-X `labels. He was a member of the Governance team, an ad-hoc employee and 1 of 2 accessibility specialists supporting VA.gov. IIRC, the reason he created 5 levels of a11y defects was so VFS teams knew how to respond to them - the lower the defect number, the more urgent the issue. I'm not sure how he landed on 5. @artsymartha68 may know more.

If the VA 508 office is only categorizing their issues on a Critical, high, medium, low scale, then maybe it's in our best interest to do the same. I am not sure how the VA 508 office would distinguish critical vs. potentially harmful. Is there a reason we're sticking with our 5 point scale? Is there concern with recategorizing defect level 0 to "critical" and level 1 to "high"? 

I'll leave it up to you and the other accessibility specialists to decide. I trust your opinion here as you are the experts.

In addition, if/when changes are made to the `508/Accessibility `label, the DraftingTemplate will be impacted, as well the Anatomy of a Staging Review Issue ticket, and lots of our operational guidance. I would suggest doing an audit of the pages we own (both VFS-facing and operational) to uncover the full list.

**Comment by briandeconinck** [comment]
@shiragoodman had better not be reading this and should be enjoying her vacation! But for others and/or when Shira gets back...

> One thing to consider, that you may want to bring to the larger a11y group, is if we really need 5 different defect levels. Trevor Pierce was responsible for creating the 508-defect-X labels. He was a member of the Governance team, an ad-hoc employee and 1 of 2 accessibility specialists supporting VA.gov. IIRC, the reason he created 5 levels of a11y defects was so VFS teams knew how to respond to them - the lower the defect number, the more urgent the issue. I'm not sure how he landed on 5. @artsymartha68 may know more.

Discussed this at today's a11y specialist sync. Where we landed: Keep `defect-0`. It's good to have available in the event of a motion/flashing light situation, even if those are extremely rare. Martha also suggests it may be useful as more health care tools move into VA.gov for flagging any issues where critical health data may not be accessible, eg. a user can't complete a critical task or make a safe decision. That's a little outside the original intent of "Potentially Harmful" but I think would be a valid use case.

I'll work on the `508/Accessibility` audit and will update the ticket.

**Comment by shiragoodman** [comment]
@it-harrison please draft a ticket for the domo & big Query pieces of this work. We will remove them from this ticket.

**Comment by briandeconinck** [comment]
@shiragoodman Just noting that I'm grabbing this from the backlog

**Comment by shiragoodman** [comment]
sounds good @briandeconinck . Sprint and epic added. I also added @it-harrison as peer reviewer.

**Comment by shiragoodman** [comment]
@briandeconinck will the new label for `508-defect-0` become `a11y-defect-0 (Potentially Harmful)` or just `a11y-defect-0`? Please clarify - thanks!

**Comment by briandeconinck** [comment]
Good question @shiragoodman! I think `a11y-defect-x (Text Description)` puts the most critical information up front for VFS teams. But it does make the labels a little more verbose and inconsistent with our other labels, and I'm not sure if it makes the Domo data feed more complicated. I think I have a slight preference for including the text description in with the label, but it's not a strong preference.

**Comment by shiragoodman** [comment]
@briandeconinck i would prefer the label without the parentheses but please confirm with Martha. This is a decision that Governance can weigh in on but I don't think we're the final decision maker here.

**Comment by briandeconinck** [comment]
@it-harrison I have all of the [draft documentation changes](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2636218438/56709+Drafts) ready to go for this. Ready for peer review.

I just want to confirm:
- The existing Domo dashboards don't currently use the `508-defect-x` labels at all, so I can change those labels whenever I want.
- The existing Domo dashboards **do** currently use the `508/Accessibility` label, so something in Domo has to happen in #58088 to keep the dashboards working as intended.

Let me know when you're ready to complete #58088 and we can coordinate the timing for the label renaming.

**Comment by it-harrison** [comment]
@briandeconinck everything looks good (I'm assuming Martha said it was okay to leave off the part with the parentheses). I will attend Analytics office hours today (5/18) and discuss making the change and will sync with you afterwards.

**Comment by artsymartha68** [comment]
Sorry this fell below the fold in my inbox. Ugh. 

Yes, I'm fine with whatever y'all think is best. I agree to leave out the parenthetical content from the label.

**Comment by shiragoodman** [comment]
@briandeconinck this has moved into the next sprint. Can you please comment with the current status? Ty!

**Comment by briandeconinck** [comment]
Status update:

- Waiting on #58088
- Reached out to other accessibility specialists to identify other repos where labels may need to be updated. Details included in the ticket body.
- [Draft change history post completed](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2645000283/56709+-+Draft+change+announcements), draft Slack message mostly completed

**Comment by shiragoodman** [comment]
@briandeconinck I thought this was included in the ticket, but we also need to update the DraftingTemplate to use the new labels. I can make the change, just let me know when I should. Also, if you could please add it to the task list and tag my name. Thank you!

**Comment by briandeconinck** [comment]
Per @it-harrison in #58088, we're ready to update labels. I will plan on updating labels Monday morning and updating Platform website docs and change history with the expectation that those changes are published Monday afternoon, then post the announcement in Slack. @shiragoodman any concerns about that sequence or the [draft change announcements](https://vfs.atlassian.net/wiki/spaces/PPST/pages/2645000283/56709+-+Draft+change+announcements)?

**Comment by shiragoodman** [comment]
@briandeconinck confirming that I am on board with the proposed approach. Please let me know if there's anything I can do to assist!

**Comment by it-harrison** [comment]
@briandeconinck looks good!

**Comment by shiragoodman** [comment]
Baseline DraftingTemplate has been updated with correct a11y labels
